<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerClass</class>
 <widget class="QWidget" name="CustomerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1473</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Покупатель</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget_23">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1461</width>
     <height>537</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="products_layout"/>
  </widget>
  <widget class="QPushButton" name="save_products_table_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>359</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Сохранить данные в отчёт</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="upd_products_table_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>359</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Обновить данные в таблице</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>550</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Параметры</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="higher_price">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>610</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="param_search_btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>660</y>
     <width>961</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Поиск изделия по параметрам</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lower_price">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>610</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="product_type">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>610</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="stone">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>610</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="metal">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>610</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>570</y>
     <width>961</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>580</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вид изделия</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>580</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Камень</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>580</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Металл</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>580</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Минимальная цена, руб</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>580</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Максимальная цена, руб</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
